test/tests.yml

```diff
@@ -213,6 +269,42 @@
     - name: vsphere_default_stable
     - name: vsphere_flatcar_stable
 
+- scenario: upgrade_containerd_external
+  initVersion: v1.29.2
+  upgradedVersion: v1.30.1
+  initKubeOneVersion: "main"
```

Comment:
Let's drop this `initKubeOneVersion` from all tests and instead update https://github.com/kubermatic/kubeone/blob/0ae1753af79abd3415f08bf907fe18f58c701ecb/test/e2e/scenario_upgrade.go#L39 to `release/v1.8`.